account_financial_report/report/abstract_report_xlsx.py

```diff
@@ -2,9 +2,11 @@
 # Author: Julien Coux
 # Copyright 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from odoo import models, fields
+from odoo import models
 import datetime
 
+from odoo.tools import format_date
```

Comment:
Shouldn't be `from odoo.tools.misc import format_date`?

Reply:
`format_date` is both in `odoo.tools` and `odoo.tools.misc` due to https://github.com/odoo/odoo/blob/98ec5df0ccea6c22024132c31010c0dac606cc27/odoo/tools/__init__.py#L10.
I have found both in the code, for instance:
* `odoo.tools`: https://github.com/odoo/enterprise/blob/c555459062af0a90296a08dad98b66180943935f/sale_subscription/models/sale_subscription.py#L14
* `odoo.tools.misc`: https://github.com/OCA/l10n-italy/blob/9ef10fa194279ad1c7e7c8502ca13cd49ff85cad/l10n_it_dichiarazione_intento/models/account_invoice.py#L6

Reply:
> due to https://github.com/odoo/odoo/blob/98ec5df0ccea6c22024132c31010c0dac606cc27/odoo/tools/__init__.py#L10.

ok, thanks for sharing it.